app/views/time_clock_punches/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<% breadcrumb :time_clock_punches %>
+
+<%= search_form_for @q, data: {"controller": "turbo_search", "turbo-frame": "results"} do |f| %>
+<div class="card">
+  <div class="card-header">
+    <h1>Time Clock</h1>
+  </div>
+  <div class="card-body">
+    <div class="row">
+      <div class="col">
+        <%= f.input :person_display_name_cont, label: 'Person' %>
+      </div>
+      <div class="col">
+        <%= f.input :start_time_gteq, label: 'From', as: :datetime %>
+      </div>
+      <div class="col">
+        <%= f.input :start_time_lteq, label: 'To', as: :datetime  %>
+      </div>
+      <div class="col">
+        <%= f.input :time_clock_period_id_eq, label: 'Periods', as: :select, collection: policy_scope(TimeClockPeriod) %>
+      </div>
+    </div>
+  </div>
+  <div class="card-footer">
+    <%= link_to "New Time Clock Punch", new_time_clock_punch_path, class: 'btn btn-success' %>
+    <%= link_to "Time Clock Periods", time_clock_periods_path, class: 'btn btn-primary' %>
```

Comment:
This should be a btn-info or something other than btn-primary

Reply:
Done